lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -19,6 +19,10 @@
 %>
 
 <%
+  disable_unenroll_cert_states = [
+    'generating',
+    'ready',
```

Comment:
the `_cert_info` method consolidates the messages into its own list of states (:facepunch: :anguished:) https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/views.py#L287